conans/build_info/model.py

```diff
@@ -0,0 +1,57 @@
+import conans
+
+
+class BuildInfo(object):
+
+    def __init__(self, name=None, number=None, started=None):
+        self.modules = []
+
+    def serialize(self):
+        return {"modules": [module.serialize() for module in self.modules],
+                "buildAgent": {"name": "Conan", "version": conans.__version__}}
+
+
+class BuildInfoModule(object):
+
+    def __init__(self):
+        # Conan package or recipe
+        self.id = ""
+        self.artifacts = []
+        self.dependencies = []
+
+    def serialize(self):
+        return {"id": self.id,
+                "artifacts": [ar.serialize() for ar in self.artifacts],
+                "dependencies": [dep.serialize() for dep in self.dependencies]}
+
+
+class BuildInfoModuleArtifact(object):
```

Comment:
When I had these code in package tools I did more things like send to artifactory and some other processing, but now, sure, only named tuples.